cmd/download.go

```diff
@@ -29,6 +29,17 @@ Download other people's solutions by providing the UUID.
 `,
 	Args: cobra.ExactArgs(1),
 	RunE: func(cmd *cobra.Command, args []string) error {
+		token, err := cmd.Flags().GetString("token")
+		if err != nil {
+			return err
+		}
+		if token != "" {
+			RootCmd.SetArgs([]string{"configure", "--token", token})
+			err := RootCmd.Execute()
+			if err != nil {
```

Comment:
I don't have very strong feelings about this, but I tend to put this on one line when there's a single return value and it is err:

```
if err := RootCmd.Execute(); err != nil {
```

Reply:
makes sense. i'll update.